src/openvslam/tracking_module.cc

```diff
@@ -179,6 +185,26 @@ Mat44_t tracking_module::track_RGBD_image(const cv::Mat& img, const cv::Mat& dep
     return curr_frm_.cam_pose_cw_;
 }
 
+void tracking_module::request_update_pose(const Mat44_t& pose) {
+    std::lock_guard<std::mutex> lock(mtx_update_pose_);
+    update_pose_is_requested_ = true;
+    request_pose_ = pose;
+}
+
+bool tracking_module::update_pose_is_requested() {
+    std::lock_guard<std::mutex> lock(mtx_update_pose_);
+    return update_pose_is_requested_;
+}
+
+bool tracking_module::finish_update_pose_request() {
+    std::lock_guard<std::mutex> lock(mtx_update_pose_);
+    if (update_pose_is_requested_) {
+        update_pose_is_requested_ = false;
+        return true;
+    }
+    return false;
+}
```

Comment:
This return value is confusing. It doesn't look like it should return anything.

Reply:
After https://github.com/OpenVSLAM-Community/openvslam/pull/79#discussion_r640587329 thread this part is not required. Changed bool return to void in [there](https://github.com/OpenVSLAM-Community/openvslam/pull/79/files#diff-9d7c6e92591d28cb8c87b3973729fddb3abd564e9feae8861851fd48dd31b2caR211-R216).